packages/run-protocol/src/proposals/committee-proposal.js

```diff
@@ -1,46 +1,7 @@
 // @ts-check
 import { E } from '@endo/far';
 
-// must match packages/wallet/api/src/lib-wallet.js
-const DEPOSIT_FACET = 'depositFacet';
-
-const { details: X } = assert;
-
-/**
- * @param {ERef<NameAdmin>} nameAdmin
- * @param {string[][]} paths
- */
-const reserveThenGetNamePaths = async (nameAdmin, paths) => {
-  /**
-   *
-   * @param {ERef<NameAdmin>} nextAdmin
-   * @param {string[]} path
-   */
-  const nextPath = async (nextAdmin, path) => {
-    const [nextName, ...rest] = path;
-    assert.typeof(nextName, 'string');
-
-    // Ensure we wait for the next name until it exists.
-    await E(nextAdmin).reserve(nextName);
-
-    if (rest.length === 0) {
-      // Now return the readonly lookup of the name.
-      const nameHub = E(nextAdmin).readonly();
-      return E(nameHub).lookup(nextName);
-    }
-
-    // Wait until the next admin is resolved.
-    const restAdmin = await E(nextAdmin).lookupAdmin(nextName);
-    return nextPath(restAdmin, rest);
-  };
-
-  return Promise.all(
-    paths.map(async path => {
-      assert(Array.isArray(path), X`path ${path} is not an array`);
-      return nextPath(nameAdmin, path);
-    }),
-  );
-};
+import { reserveThenDeposit } from './utils.js';
```

Comment:
how about `nameHubAux.js` or `proposalUtils.js`? `utils.js` says very little about what's inside.

Reply:
It's already `proposals/utils.js`.  Is that enough, or would you prefer `proposals/proposalUtils.js`?

Reply:
> Is that enough

ok